share/actionsmap.yml

```diff
@@ -37,6 +37,32 @@ _global:
     authentication:
         api: ldap_admin
         cli: null
+    arguments:
+        -h:
+            full: --help
+            help: Show this help message and exit
+        --version:
+            help: Display YunoHost packages versions
+            action: callback
+            callback:
+                method: yunohost.utils.packages.ynh_packages_version
+                return: true
+        --output-as:
+            help: Output result in another format
+            choices:
+               - json
+               - plain
+               - none
+        --debug:
+            help: Log and print debug messages
+            action: store_true
+        --quiet:
+            help: Don't produce any output
+            action: store_true
+        --timeout:
+            help: Number of seconds before this command will timeout because it can't acquire the lock (meaning that another command is currently running), by default there is no timeout and the command will wait until it can get the lock
+            type: int
+
```

Comment:
Hmmm I'm confused about this ... I understand that you need some declarative way of finding about these for the zsh autocompletion, but I doubt that `moulinette` really does handle these ? It's gonna be pretty confusing when debugging the action map later ... Also I see that you have a `GLOBAL_ARGUMENTS` which kinda hard-code those in `generate_zsh_completion.py` so i'm confused x_X

Reply:
Hey, thanks for reviewing. This section was in the actionsmap before and it was removed. I didn't test with `moulinette` actually. I let the `GLOBAL_ARGUMENTS` in the code in case the section is missing in actionsmap, which can happen if the actionsmap file is still an old version. I don't know what's best... I think having the arguments in the `_global` section in actionsmap makes is easier to update them because they are all together. But if it is not working with `moulinette` we can remove the section.